project_stage_closed/migrations/2.0.0/post-remove-project-task-type-closed-field.py

```diff
@@ -0,0 +1,16 @@
+import logging
```

Comment:
No need to over-log. Odoo already mentions about executing migration script, but I let it to your taste.